website/server/models/user/schema.js

```diff
@@ -68,6 +68,7 @@ let schema = new Schema({
     timestamps: {
       created: {$type: Date, default: Date.now},
       loggedin: {$type: Date, default: Date.now},
+      updated: {$type: Date, default: Date.now},
```

Comment:
for this one, instead of setting it manually can we use the `pre('save'` and `pre('update'`user hooks that we already use to increase the user version field?

Reply:
The problem with that is it'll cause the updated timestamp to change on any number of updates to the user that don't indicate they've interacted with the site or app. For instance, if they receive a notification, `pre` hooks will fire, but that happens without any interaction whatsoever on their part.

Reply:
Hmm right.. I still don't like having to set it manually, it's very error prone